src/input.rs

```diff
@@ -125,6 +125,12 @@ pub fn get_char_pressed() -> Option<char> {
     context.chars_pressed_queue.pop()
 }
 
+pub fn get_char_pressed_ui() -> Option<char> {
```

Comment:
Could you make it `pub(crate)`? 
UI will consume the entire queue anyway, so no need to make this function available in the public API